### PR TITLE
Updated machine_types.go for i3 instances

### LINKED ISSUE #2006: Update aws/aws-sdk-go to support new instance types
When trying to create an instance group with i3 instance groups I receive. It looks like this would simply require an update to `aws-sdk-go`

```W0227 12:40:23.642261   12478 executor.go:109] error running task "LaunchConfiguration/els-data-node.ca-prod.kube.granduke.net" (9m58s remaining to succeed): instance type not handled: "i3.2xlarge"```

### PR DESCRIPTION
**Pull request by geojaz**
Closes #2006.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2007)

### CODE REVIEW

File: upup/pkg/fi/cloudup/awsup/machine_types.go [hunk 1]
```diff
@@ -396,6 +396,50 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
...
+		ECU:            104,
+		Cores:          32,
+		EphemeralDisks: []int{1900, 1900, 1900, 1900},
+	},
+	{
+		Name:           "i3.16xlarge",
+		MemoryGB:       488,
+		ECU:            208,
+		Cores:          64,
+		EphemeralDisks: []int{1900, 1900, 1900, 1900},
```
Comment (yissachar) [inline upup/pkg/fi/cloudup/awsup/machine_types.go]: Shouldn't there be 8 of these?

**Comment by yissachar** [comment]
How did you get the ECU units? They look a little different than the numbers I'm seeing.

**Comment by geojaz** [comment]
I see some discrepancy in the ECU numbers between the [AWS marketing](https://aws.amazon.com/ec2/pricing/on-demand/) materials  and the console. I didn't notice them before, but I chose the numbers from the console with no good reason for or against. Anyone else want to clarify?

screen shot 2017-02-28 at 6 20 16 am

File: upup/pkg/fi/cloudup/awsup/machine_types.go (hunk 1 above)
Comment (geojaz) [inline upup/pkg/fi/cloudup/awsup/machine_types.go]: O:) thanks!

**Comment by justinsb** [comment]
I also noticed the ECU discrepancy, but the numbers are close enough IMO, and we aren't actually using the ECUs currently.  I'm excited to try out this instance type (well, the smaller ones!).

**Pull request merged**